umh.docs.umh.app/content/en/docs/production-guide/installation/flatcar-installation-bare-metal.md

```diff
@@ -13,9 +13,7 @@ aliases:
 
 Here is a step-by-step guide on how to deploy the UMH stack on
 [Flatcar Linux](https://www.flatcar.org/), a Linux distribution designed for
-container workloads, with high security and low maintenance.
-
-This is a good option if you want to deploy the UMH stack on edge devices or IPCs.
+container workloads with high security and low maintenance.
```

Comment:
This will leverage the UMH Device and Container Infrastructure, which is used in this example as the community edition. During the installation, we will download and boot on your device / on our VM from the bootloader iPXE. iPXE, if donwloaded further below, is pre-configured and tailored to the UMH. First, it will guide you through an installation assistant. Then it will first load flatcar from the latest LTS version (current-2023) and boot into it. This system is called flatcar-0. Then it will install flatcar to disk. Upon automatic rebooting, it will boot into the just installed flatcar instance called flatcar-1, which will automatically install the UMH.

Reply:
I added after checking grammar